addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -23,6 +24,15 @@ var SmsWidget = FieldText.extend({
         this.nbrChar = 0;
         this.nbrSMS = 0;
         this.encoding = 'GSM7';
+        this.enable_emojis = !!this.nodeOptions.enable_emojis;
```

Comment:
The nodeOptions can stay "enable_emojis" but you can camelCase the widget property to enableEmojis.
In other words:
`this.enableEmojis = !!this.nodeOptions.enable_emojis;`